scripts/jenkins/cloud/ansible/crowbar-update.yml

```diff
@@ -47,25 +47,26 @@
               delegate_to: "{{ target_host }}"
           vars:
             inventory_hostname: "{{ target_host }}"
-          loop: "{{ [cloud_env]+groups['cloud_virt_hosts'] }}"
+          loop: "{{ groups['cloud_virt_hosts'] }}"
           loop_control:
             loop_var: target_host
 
         - include_role:
-            name: reboot_node
+            name: crowbar_setup
           vars:
-            reboot_target: deployer
-          when:
-            - update_after_deploy
-            - reboot_after_deploy
+            qa_crowbarsetup_cmd: onadmin_rebootcloud
```

Comment:
We have a jenkins option to request reboot_after_deploy. So maybe there could be not `update_after_deploy` && `reboot_after_deploy` but only `reboot_after_deploy`(since for some reason u just want to install an update and do not reboot cloud nodes or investigate an issue before the reboot.

Reply:
Just to clarify, checking `update_after_deploy` in this playbook is redundant, since the playbook is only executed when `update_after_deploy` is set (see [here](https://github.com/SUSE-Cloud/automation/blob/master/jenkins/ci.suse.de/pipelines/openstack-crowbar.Jenkinsfile#L226) and [here](https://github.com/SUSE-Cloud/automation/blob/master/scripts/jenkins/cloud/manual/lib.sh#L253)).
I guess using `reboot_after_deploy` to skip rebooting the nodes (cloud/admin) makes sense in this case, but only because we're no longer relying on nodes coming back from reboot to apply the chef configuration (which turns out was insufficient in the first place, hence this PR). Let me bring that option back.